runTests.py

```diff
@@ -43,6 +43,13 @@ def processCLIArgs():
         action="store_true",
         help="Don't run tests, just try to make them.",
     )
+    parser.add_argument(
+        "--only-functions",
```

Comment:
`--only-functions` accepts multiple values so it will eat up `src/test/expressions` too. You can specify multiple functions this way:
```
./runTests.py --only-functions sum src/test/expressions --only-functions sum sin cos tan
```
This also works
```
./runTests.py --only-functions sum sin -j1 src/test/expressions
```

> Can there be an error if the function isn't found?

Done

Reply:
Oh weird. I guess as long as this is written down wherever the instructions are for running the tests it's fine.